src/utils/azure.ts

```diff
@@ -66,6 +69,66 @@ export async function getStorageConnectionString(context: IStorageAccountWizardC
     };
 }
 
+export async function getEventHubsConnectionString(context: IEventHubsConnectionWizardContext & ISubscriptionContext): Promise<IResourceResult> {
+    const client: EventHubManagementClient = await createEventHubClient(context);
+    const resourceGroupName: string = getResourceGroupFromId(nonNullValue(context.eventHubsNamespace?.id));
+    const namespaceName: string = nonNullValue(context.eventHubsNamespace?.name);
+
+    const authRulesIterable = client.namespaces.listAuthorizationRules(resourceGroupName, namespaceName);
+    const authRules: AuthorizationRule[] = await uiUtils.listAllIterator(authRulesIterable);
+
+    let authRule: string;
+    if (!authRules.length) {
```

Comment:
I think this probably works for now, but I want to revisit this logic in a future PR before we merge the final into main.  I have an open issue here for tracking: https://github.com/microsoft/vscode-azurefunctions/issues/3445